_projects/2022/100481925/100481925.Rmd

```diff
@@ -0,0 +1,1481 @@
+  </figcaption>
+  </p>
+</div>
+
+As for the graph, the author used Tableau to build a *bump chart* that includes interactive elements when hovering over the dots; but for the purposes of this report, those elements will not be taken into account: only what is seen in the screenshot above is considered. 
+
+Bump charts are useful for exploring changes in rank over time across different categories. That is, they usually do not show the actual values of the categories -which in this case would be the money spent by each generation for each spending category-, but the positions in the ranking. In this case, the ranking jumps in 14 spending categories are shown, according to the birth year range from the Silent generation (the oldest) to Generation Z. 
+
+At first glance, this and, in general, all bump charts show three variables in an easy-to-understand way: category, time and position. That's especially true in cases of charts where *the only important thing* is the ranking. For example, this image from [The Washington Post](https://www.washingtonpost.com/news/wonk/wp/2015/02/27/americas-favorite-dog-breeds-for-the-past-two-decades-ranked/) from 2015 shows the "favorite" dog breeds, according to the American Kennel Club, where the three important variables are: breed of dog, year, and position in the ranking. 
+
+<div>
+  <p>
+  <center><img src="https://www.washingtonpost.com/blogs/wonkblog/files/2015/02/Dogchangev4.png?itid=lk_inline_manual_3" width="100%">
+  <figcaption>
+    <span class="caption"></span>
+    <i class="photo-credit">Article published by Philip Bump in The Washington Post: https://www.washingtonpost.com/news/wonk/wp/2015/02/27/americas-favorite-dog-breeds-for-the-past-two-decades-ranked/</i>
+  </figcaption>
+  </p>
+</div>
+
+However, there is another important piece of information in Preethi Lodha's graph, which is the dollar amount spent by each generation for each expenditure category. This is shown within each point and is undoubtedly important if one wants to know how spending is distributed; however, that oscillation is not represented and can be misleading. This point will be discussed later.
+
+## What can be seen here? Encodings, channels and strengths
+There are 14 spending categories represented on the Y axis and 5 generations or age ranges showed on the X axis, above the graph and below the title and subtitle, from the Silent generation (the oldest) to the Z generation.
+
+According to Cleveland (1985), graphs encode data in distinct elements, such as symbols, colors, position, etc. When we study a graphic we perform several mental-visual tasks to extract quantitative information. Some of the basic judgements we perform to decode quantitative information are 1) angle, 2) area, 3) color hue, 4) color saturation, 5) density, 6) length, 7) position along a common scale, 8) position along identical, nonaligned scales, 9) slope, 10) volume (Cleveland, 1985).
```

Comment:
There is a special syntax to add citations to R Markdown documents that not only generates links to the references, but also generates the reference list automatically at the end. See [the documentation](https://rstudio.github.io/distill/citations.html) please. It's a bit more of work but it's worth it.